pulpcore/doc_requirements.txt

```diff
@@ -1 +1 @@
--e git+https://github.com/limdauto/drf_openapi.git@54d24fb#egg=drf_openapi
```

Comment:
I don't think we need this file anymore, since this was only separate so the user can install it separately from a pinned git commit

These docs should be removed: https://github.com/pulp/pulp/blame/3.0-dev/docs/contributing/3.0-development/rest-api.rst#L473

An associated devel change should be made to remove doc_requirements here: https://github.com/pulp/devel/blob/3.0-dev/ansible/roles/dev/tasks/main.yml#L97

There also needs to be a jjb change- I've opened a PR of it here: https://github.com/pulp/pulp-ci/pull/505